README.md

```diff
@@ -1512,6 +1515,7 @@ Table of Contents
   * [RevenueCat](https://www.revenuecat.com/) — Hosted backend for in-app purchases and subscriptions (iOS and Android). Free up to $10k/mo in tracked revenue.
   * [vatlayer](https://vatlayer.com/) — Instant VAT number validation and EU VAT rates API, free 100 API requests/month
   * [Freecurrencyapi.net](https://freecurrencyapi.net/) — Free currency conversion and exchange rate data API. 10 requests/hour without an API key, 50 000 requests per month when you register for free.
+  * [Stripe](https://stripe.com) - A free payment and checkout API for cards, bank transfers with 3d secure authentication and instant payouts.
```

Comment:
I really am not sure about how free stripe is, you cannot do anything with it without paying - syaing the API is free but processing any payment incurs a cost does not make the API free

can you tell me what you found was free with it? Like something actually useful thats free

Reply:
I don't think there's something free it does cost a small fee per transaction. But as a service it's free to use (no monthly or annual payments needed).

Reply:
Free offering has to be useful for this list I think